### PR TITLE
Fix typos found by codespell

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
I have tried to leave out vendored 3rd party files, such as [`src/csync`](https://github.com/nextcloud/desktop/tree/master/src/csync).

### CODE REVIEW

**Comment by mgallien** [comment]
@DimitriPapadopoulos thanks a lot
could you please rebase on the latest work in the `master` branch ?
I would like to see those changes be merged sooner rather than later

**Comment by DimitriPapadopoulos** [comment]
@mgallien Rebased against _master_.

File: src/libsync/vfs/cfapi/shellext/cfapishellintegrationclassfactory.h [hunk 1]
```diff
@@ -13,7 +13,7 @@
  */
 
 #pragma once
-#include <unknwn.h>
+#include <unknown.h>
```
Comment (mgallien) [inline src/libsync/vfs/cfapi/shellext/cfapishellintegrationclassfactory.h:16]: this is breaking compilation on windows
can you remove this change ?

File: theme/colored/Nextcloud-macOS-icon.svg
Comment (mgallien) [inline theme/colored/Nextcloud-macOS-icon.svg:1]: I am not seeing the changes of this file via github web UI
maybe just remove the changes form this file

**Review by mgallien — CHANGES_REQUESTED** [review]
thanks again for all the fixes (especially those in the user documentation)
see my 2 comments

File: theme/colored/Nextcloud-macOS-icon.svg
Comment (DimitriPapadopoulos) [reply theme/colored/Nextcloud-macOS-icon.svg:1]: Ooops... That was not intended. I'll undo changes to SVG files. I don't know how it happened.

File: src/libsync/vfs/cfapi/shellext/cfapishellintegrationclassfactory.h (hunk 1 above)
Comment (DimitriPapadopoulos) [reply src/libsync/vfs/cfapi/shellext/cfapishellintegrationclassfactory.h:16]: Sorry about that. Will undo. It's extremely complex to view such changes across many files, tools such as `gitg` just cannot cope.

Comment (DimitriPapadopoulos) [reply src/libsync/vfs/cfapi/shellext/cfapishellintegrationclassfactory.h]: The `<unknwn.h>` change was an error on my side.

File: theme/colored/Nextcloud-macOS-icon.svg
Comment (DimitriPapadopoulos) [reply theme/colored/Nextcloud-macOS-icon.svg:1]: And this was the result of a `git merge` that went wrong for some reason. But it's been reverted.

**Comment by mgallien** [comment]
@DimitriPapadopoulos absolutely no worries
the review is there to catch those kind of issues
by the way let me say it again, thanks a lot for your contribution
that is going to fix typos not only in the code but in the user documentation, that is really valuable

**Pull request merged**